### ISSUE TITLE
Setting force_timeout with a non-numeric value shouldn't be allowed in sssd.conf

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1625]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1621
- Created at 2012-11-08 11:23:48 by [jhrozek](https://pagure.io/user/jhrozek)
- Closed as Duplicate
- Assigned to nobody
---

https://bugzilla.redhat.com/show_bug.cgi?id=874494 (*Red Hat Enterprise Linux 6*)

    Description of problem:
    It is allowed to set non-numeric values to force_timeout parameter in
    sssd.conf.
    
    Version-Release number of selected component (if applicable):
    sssd-1.9.2-7.el6.x86_64
    
    How reproducible:
    Always
    
    Steps to Reproduce:
    1. Set force_timeout = TRUE in DOMAIN Section of sssd.conf file
    2. Restart sssd service.
    3. Send SIGSTOP to process sssd_be
       # kill -s SIGSTOP <PID>
    
    4. Monitor sssd.log file for the messages.
       # tail -f /var/log/sssd.log
    
    Actual results:
    The sssd service restarts successfully and sssd_be gets killed in 60 secs.
    Results are similar to the "force_timeout" default behaviour.
    
    Expected results:
    An error should be shown while starting or restarting the sssd service.
    Non-numeric values shouldn't be allowed while setting force_timeout parameter
    in sssd.conf.

### Comments

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-11-08 15:34:51](https://pagure.io/SSSD/sssd/issue/1621#227597)**

Fields changed

blockedby:  => 
blocking:  => 
coverity:  => 
design:  => 
design_review:  => 0
feature_milestone:  => 
fedora_test_page:  => 
milestone: NEEDS_TRIAGE => SSSD Deferred
testsupdated:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2012-11-09 01:55:15](https://pagure.io/SSSD/sssd/issue/1621#227598)**

Fields changed

blocking:  => [https://fedorahosted.org/sssd/ticket/416 416]

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2015-04-24 16:01:13](https://pagure.io/SSSD/sssd/issue/1621#227599)**

Closing as duplicate of #2269

changelog:  => 
mark:  => 0
review:  => 0
selected:  =>

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2015-04-24 16:02:00](https://pagure.io/SSSD/sssd/issue/1621#227600)**

Fields changed

resolution:  => duplicate
status: new => closed

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-24 14:36:07](https://pagure.io/SSSD/sssd/issue/1621#227601)**

**Metadata Update from @jhrozek**:
- Issue marked as blocked by: #416
- Issue set to the milestone: SSSD Patches welcome
